src/models/thread.ts

```diff
@@ -709,7 +709,7 @@ export class Thread extends ReadReceipt<ThreadEmittedEvents, ThreadEventHandlerM
      * Return last reply to the thread, if known.
      */
     public lastReply(
-        matches: (ev: MatrixEvent) => boolean = (ev): boolean => ev.isRelation(RelationType.Thread),
+        matches: (ev: MatrixEvent) => boolean = (ev): boolean => ev.isRelation(THREAD_RELATION_TYPE.name),
```

Comment:
I think this was just broken and was hardcoded to the stable prefix, but the tests use the unstable prefix (because it's stored a s a singleton so once one test set it unstable, the rest will break unless they also use unstable).